misaligned.c

```diff
@@ -8,6 +8,8 @@ int printColorMap() {
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
             printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            if(i!=j)

```

Comment:
This is not functionality, but a bug to make the test fail?
Restore the functionality and strengthen the test to catch the existing bug (0-based color numbers). Don't invent bugs to fail weak tests!